data/accounts/index.js

```diff
@@ -343,4 +343,28 @@ export default [
     image: '/logos/accounts/stakeDAO.svg',
     environment: 'mainnet',
   },
+  {
+    address: '0xB8D85A5e0c5e3354d35d576818B57AEd0300dE16',
+    name: 'Prime Protocol Moonbeam',
```

Comment:
```suggestion
    name: 'Prime Protocol',
```
The chain is already mentioned on axelarscan, so just the protocol name should be sufficient everywhere